src/devtools/client/webconsole/utils/autocomplete.ts

```diff
@@ -149,8 +149,31 @@ function getBinding(name: string, scope: Scope) {
 function getBindingNames(scope: Scope): string[] {
   return scope.bindings.map(b => b.name);
 }
+function getGlobalActor(scopes: Scope) {
+  let globalActor = scopes;
+
+  while (globalActor.parent) {
+    globalActor = globalActor.parent;
+  }
+
+  return globalActor;
+}
 function getGlobalVariables(scopes: Scope) {
```

Comment:
Should we maybe trigger this `onMount` or so, as it's done only once, it's very likely by the time it finishes fetching children the user hasn't even started using the autocomplete.

Reply:
this is interesting because we should probably do it on mount, but we might also need to do it when we pause because each pause could have new globals

Reply:
Exactly. We could be more responsive and start fetching the global's children once paused, but I'd rather take it more conservatively for the first pass.